internal/controllers/analyzer/analyzer.go

```diff
@@ -372,3 +394,38 @@ func (a *Analyzer) removeVulnerabilitiesByTypes() *analysis.Analysis {
 
 	return a.analysis
 }
+
+// setUpdateHashWarnings checks for hashes generated in older formats but that are still valid. If one of
+// these hashes are found, a warning will be showed informing the user to update the outdated hash.
+// TODO: Remove setUpdateHashWarnings before release v2.10.0
+// nolint
+func (a *Analyzer) setUpdateHashWarnings() {
+	isPrintDepreciationMsg := true
+
+	vulnerabilities := a.analysis.AnalysisVulnerabilities
+	for vulnIndex := range vulnerabilities {
+		for _, deprecatedHash := range vulnerabilities[vulnIndex].Vulnerability.DeprecatedHashes {
+			for _, configHash := range a.getAllConfigHashes() {
```

Comment:
I think that would be good to create a variable with the results of `a.getAllConfigHashes()` to avoid recompute this value very time.

Reply:
Well observed, done!